tests/integration_tests/data_steward/cdr_cleaner/cleaning_rules/store_pid_rid_mappings_test.py

```diff
@@ -0,0 +1,116 @@
+from datetime import datetime
+import os
+
+import mock
+
+from app_identity import get_application_id
+from common import PID_RID_MAPPING, PRIMARY_PID_RID_MAPPING
+from resources import fields_for
```

Comment:
looks good!
nit: there are 2 unused Imports in the test script.

Reply:
@ksdkalluri nice catch!  Just removed them.  I'll wait until tests pass, and then merge with your approval.